### PR TITLE
Fix deprecated sf3.4

### PR DESCRIPTION
**Pull request by Barbarians**

### CODE REVIEW

File: Form/ConfiguredTypeType.php [hunk 1]
```diff
@@ -20,9 +21,9 @@ class ConfiguredTypeType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('name')
+            ->add('blockPrefix')
```
Comment (burahimu) [inline Form/ConfiguredTypeType.php:24]: Why did you rename `name` to `blockPrefix` ?

Comment (Barbarians) [reply Form/ConfiguredTypeType.php:24]: see this commit : https://github.com/IDCI-Consulting/ExtraFormBundle/commit/5b9941e140d0c2f7156776e013a8b6bfe59a69fc#diff-c9cc67084459152c0a52aaa1da13bd16R17

Comment (burahimu) [reply Form/ConfiguredTypeType.php:24]: Nice catch :+1:

**Review by burahimu — APPROVED** [review]

**Comment by Barbarians** [comment]
Up !!!!

**Review by konandrum — APPROVED** [review]
